mesonbuild/utils/universal.py

```diff
@@ -1103,23 +1129,7 @@ def quote_arg(arg: str) -> str:
         if arg and not _find_unsafe_char(arg):
```

Comment:
Is that still needed? Maybe it should be moved to cmd_quote() ?

Reply:
yes, it prevent quoting when not needed. Indeed, I moved it to cmd_quote, as there are no reason to add quotes when not needed in the response file.